README.md

```diff
@@ -36,6 +36,7 @@ You can define key bindings to send JavaScript codes to REPL like below:
             (define-key js-mode-map (kbd "C-x C-e") 'nodejs-repl-send-last-expression)
             (define-key js-mode-map (kbd "C-c C-j") 'nodejs-repl-send-line)
             (define-key js-mode-map (kbd "C-c C-r") 'nodejs-repl-send-region)
+            (define-key js-mode-map (kbd "C-c C-c") 'nodejs-repl-send-buffer)
```

Comment:
Can you also add the same line to the comment in nodejs-repl.el?

Reply:
Done